src/stcal/resample/resample.py

```diff
@@ -802,7 +802,7 @@ def add_model(self, model):
         add_image_kwargs = {
             'exptime': model["exposure_time"],
             'pixmap': pixmap,
-            'scale': iscale,
+            'scale': self.pixel_scale_ratio,
```

Comment:
If `scale` is now scale ratio, at least add a comment to it to avoid confusion in the future.